mcs/class/System.Security/Makefile

```diff
@@ -8,7 +8,7 @@ MONO_SECURITY=Mono.Security
 endif
 
 LIBRARY = System.Security.dll
-LIB_REFS = secxml/System bare/System.Xml $(MONO_SECURITY)
+LIB_REFS = secxml/System bare/System.Xml System.Numerics $(MONO_SECURITY)
```

Comment:
The `KeyInfoX509Data` uses `BigInteger` from there: https://github.com/dotnet/corefx/blob/master/src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/KeyInfoX509Data.cs#L164.